app/src/main/java/com/kickstarter/features/pledgedprojectsoverview/ui/PledgedProjectsOverviewScreen.kt

```diff
@@ -72,6 +74,8 @@ fun PledgedProjectsOverviewScreen(
     errorSnackBarHostState: SnackbarHostState,
     ppoCards: LazyPagingItems<PPOCardDataMock>,
     totalAlerts: Int = 0,
+    onCardClick: () -> Unit,
+    onProjectPledgeSummaryClick: (backingDetailsUrl: String) -> Unit,
```

Comment:
Added a new `onProjectPledgeSummaryClick` lambda for just the pledge summary tap area per the ticket AC. The `onCardClick` lambda is for the tap area of the entire card, not sure if it's going to end up being used?